### PR TITLE
Fix incidence calculation

### PR DESCRIPTION
**Pull request by halfhorst**
This will convert the incidence we get from get_draws (total incidence) to the incidence we want/expect, the hazard rate.

### CODE REVIEW

File: src/vivarium_inputs/core.py [hunk 1]
```diff
@@ -308,7 +308,23 @@ def _get_incidence(entities, location_ids):
...
+    incidence = measure_data[correct_measure & correct_years]
+    prevalence = _get_prevalence(entities, location_ids)
+
+    draw_columns = [col for col in incidence.columns if col.startswith('draw_')]
+
+    incidence.set_index(['age_group_id', 'location_id', 'cause_id',
+                             'sex_id', 'year_id'], inplace=True)
+    prevalence.set_index(['age_group_id', 'location_id', 'cause_id',
+                              'sex_id', 'year_id'], inplace=True)
+
```
Comment (michaella21) [inline src/vivarium_inputs/core.py]: I'm not 100% sure, but pandas set_index probably doesn't stay until you re-define like `incidence=incidence.set_index...`

Comment (halfhorst) [reply src/vivarium_inputs/core.py]: it does only because I have inplace=True there

Comment (michaella21) [reply src/vivarium_inputs/core.py]: ahhhhh I missed that

**Comment by collijk** [comment]
We need to make sure this doesn't break anything.  Can you run the smoke tests in ceam_experiments?

**Comment by halfhorst** [comment]
will do

**Comment by halfhorst** [comment]
breaks on 'cause_id' sometimes, don't know what guarantees I have around what columns will be around. Need to be able to line up the quotient between the incidence and prevalence dataframes. Looking in to it

**Pull request merged**